src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -82,7 +83,7 @@ private ResponseEntity postEventInternal(final String eventTypeName,
             final JSONArray eventsAsJsonObjects = new JSONArray(eventsAsString);
 
             final int eventCount = eventsAsJsonObjects.length();
-            eventTypeMetrics.reportSizing(eventCount, eventsAsString.length());
+            eventTypeMetrics.reportSizing(eventCount, eventsAsString.getBytes(Charsets.UTF_8).length);
```

Comment:
@lmontrieux 
During validation I found one problem. It's not related to your fix, your fix works pretty good.
The thing is that we update statistics before publishing of events. So it can be that events are rejected because of validation problem but we anyway include event sizes to our metrics.
So for me it makes sense to include to statistics only the events that were actually published. What do you think?